includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -3352,7 +3348,6 @@ class AMP_Allowed_Tags_Generated {
 							4,
 						),
 					),
-					'bento' => true,
```

Comment:
Rather than just removing the `bento` flag here, the spec generator should rather be removing this entire array item for `amp-facebook-page 1.0`. I believe any spec that includes `bento_supported_version` should be skipped, in particular:

https://github.com/ampproject/amphtml/blob/2e2cdc18d1fbc196b16eced63c9ab7415fc1e77f/extensions/amp-facebook/validator-amp-facebook.protoascii#L1-L19

Humm, then we'll need to exclude any tag specs that have a `requires_extension`/`requires_condtition` that reference such a spec:

https://github.com/ampproject/amphtml/blob/2e2cdc18d1fbc196b16eced63c9ab7415fc1e77f/extensions/amp-facebook/validator-amp-facebook.protoascii#L84-L100